app/components/Welcome.js

```diff
@@ -3,10 +3,21 @@ import { Button, Fade } from 'reactstrap'
 import { Link } from 'react-router-dom'
 import PropTypes from 'prop-types'
 import Spinner from './Spinner'
+import { SUPPORTED_LANGS } from '../constants/i18n'
 
 import { setNotFirstTimeFlag } from '../../utils/api.electron'
 
 export default class Welcome extends Component {
+  constructor (props) {
+    super()
+    this.state = { language: 'en' }
```

Comment:
If i don't create the state with a fixed value if gives me an error and throws undefined when referencing the state

Reply:
what about using
`
this._defaults = {
      language: getDefaultLanguage()
    }
` like settings.js?